tox.ini

```diff
@@ -20,6 +20,17 @@ commands =
     coverage report -m --fail-under 99
 
 
+# Develop test env to run tests against securesystemslib's main branch
+# Must to be invoked explicitly with, e.g. `tox -e with-sslib-main`
+[testenv:with-sslib-main]
+commands_pre =
+    pip install --force-reinstall git+https://github.com/secure-systems-lab/securesystemslib.git@main#egg=securesystemslib[crypto,pynacl]
+
+commands =
+    coverage run tests/runtests.py
+    coverage report -m
```

Comment:
Should this also include the `--fail-under` flag?

Reply:
I think it's fine without.